package/kernel/linux/modules/netsupport.mk

```diff
@@ -1434,6 +1434,23 @@ endef
 $(eval $(call KernelPackage,inet-diag))
 
 
+define KernelPackage/xdp-sockets-diag
+  SUBMENU:=$(NETWORK_SUPPORT_MENU)
+  TITLE:=PF_XDP sockets monitoring interface support for ss utility
+  KCONFIG:= \
+	CONFIG_XDP_SOCKETS=y \
```

Comment:
This `=y` builds-in `XDP_SOCKETS` into every kernel on the buildbots, via `ALL_KMODS`. You should state the size difference this makes, and why it is justified in the commit message.
Otherwise, you could add `XDP_SOCKETS` as an OpenWrt `Config-kernel` option, and depend on that in the package. Example: https://github.com/openwrt/openwrt/commit/45d541bb409355f090b971d96ebebd8610ef84a7

Reply:
done